src/status_im/ui/components/list_selection.cljs

```diff
@@ -30,17 +30,20 @@
          :options     (message-options message-id old-message-id text)
          :cancel-text (i18n/label :t/message-options-cancel)}))
 
+(defn- platform-web-browser []
+  (if platform/ios? :t/browsing-open-in-ios-web-browser :t/browsing-open-in-android-web-browser))
```

Comment:
no desktop?

Reply:
No browser on desktop for now? And not sure there would be a distinction between browsers as on mobile. But good point.